src/sentry_utils.c

```diff
@@ -462,6 +462,8 @@ sentry__iso8601_to_usec(const char *iso)
     tm.tm_sec = s;
 #ifdef SENTRY_PLATFORM_WINDOWS
     time_t time = _mkgmtime(&tm);
+#elif defined(SENTRY_PLATFORM_PS)
+    time_t time = mktime(&tm); // TODO is this correct?
```

Comment:
Is the question directed at me? `mktime()` assumes `tm` contains a local-time offset (whatever that is). It is equivalent to `timelocal()`. So, since that part of the code parses an ISO8601 string that always ends with `Z`, you will potentially introduce a round-trip error between it and `sentry__usec_time_to_iso8601()`. The function is currently only used in tests, by the way.

If you want to be sure, you must find some way to access the local time offset (but I am not sure what POSIX interfaces PS exposes, given that `timegm` isn't available).

Of course, if PS guarantees to only have local-time offsets of 0, at that level, then you're good too.

Reply:
thx for the feedback. I've had to move the code to the downstream repo to make it work properly.